### DISCUSSION TITLE
nokogiri.bundle' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64'))

### DISCUSSION THREAD

**Discussion by adiittriyadi** [sparklemotion/nokogiri#3048] [General Q&A]
Hi guys i facing this issue already try many ways but still happened anyone have idea how to solved this issue?
`nokogiri.bundle' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64'))`
Im running on MacOS Sonoma M1 Pro

**Comment by flavorjones** [comment]
@adiittriyadi This error message indicates that you're trying to use a nokogiri gem that was compiled for `x86_64` architecture (possibly the precompiled native gem). Without more information about how your system got into this state, it's hard to give you advice.

If you're using bundler, what's the output from `bundle env`?

**Reply by adiittriyadi** [reply]
btw im forgot update this question , already solved im just install nokogiri for arm64 hehe `arch -arm64 gem install nokogiri -v '1.12.5'`
